src/pages/Location/Location.js

```diff
@@ -65,14 +65,14 @@ const LocationComponent = () => {
   const councilLabelRedux = useSelector((stateRedux) => stateRedux.mapData.councilLabel);
   const progressRedux = useSelector((stateRedux) => stateRedux.sharedData.progress);
   const myCoverCropListLocationRedux = useSelector((stateRedux) => stateRedux.sharedData.myCoverCropListLocation);
-  const regionShorthand = useSelector((stateRedux) => stateRedux.mapData.regionShorthand);
   const accessTokenRedux = useSelector((stateRedux) => stateRedux.userData.accessToken);
   const userFieldRedux = useSelector((stateRedux) => stateRedux.userData.field);
 
   // useState vars
+  const [selectedRegion, setRelectedRegion] = useState();
```

Comment:
seems like this should be using the redux var

Reply:
This is only used in this file and i removed the redux var per the cleanup unused redux state ticket. do we want to add that back in?

Reply:
yeah you're right, sorry